src/lib/apmJson.ts

```diff
@@ -7,34 +7,26 @@ import {
 import log from 'electron-log';
 import { readJsonSync, writeJsonSync } from 'fs-extra';
 import path from 'path';
-
-/**
- * Returns the path of `apm.json`.
- *
- * @param {string} instPath - An installation path
- * @returns {string} The path of `apm.json`
- */
-function getPath(instPath) {
-  return path.join(instPath, 'apm.json');
-}
+import { ApmJsonObject } from '../types/apmJson';
+import { PackageItem } from '../types/packageItem';
 
 /**
  * Gets the object parsed from `apm.json`.
  *
  * @param {string} instPath - An installation path
  * @returns {object} An object like `package.json`
  */
-function getApmJson(instPath) {
+function getApmJson(instPath: string) {
   try {
     const value = readJsonSync(getPath(instPath));
     if (typeof value === 'object') {
-      return value;
+      return value as ApmJsonObject;
     } else {
       throw new Error('Invalid apm.json.');
     }
   } catch (e) {
     if (e.code !== 'ENOENT') log.error(e);
-    return { dataVersion: '3', core: {}, packages: {} };
+    return { dataVersion: '3', core: {}, packages: {} } as ApmJsonObject;
```

Comment:
```suggestion
    return { dataVersion: '3', core: {}, packages: {} };
```

It may be better not to put `as` on objects that are entered manually.